blog/_posts/2021-08-20-Aria-Lanea-Blog.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: "Planning for the Future of MABE2"
+date: 2021-08-20
+author: Aria Killibrew Bruehl and Lanea Rohan
+---
+
+## Outline
+
+# :soon: Planning for the Future of MABE2: A Summer of Documentation and Testing
+​
+Hello! This summer we (Lanea Rohan, '22 Grinnel College, and Aria Killebrew Bruehl,
+'23 Reed College) participated in the WAVES workshop through Michigan State University.
+We dedicated our summer to documenting and testing MABE2, an open-source research platform that provides accessible tools for conducting evolutionary computation and digital evolution research.
+​
+This project was guided by our wonderful mentors [Acaica Ackles](https://alackles.github.io/) (documentation) and [Austin Ferguson](http://fergusonaj.com/) (testing).
+
+​
+
+## :interrobang: Why Document and Test Code
+​
+The [Digital Evolution Lab](https://devolab.org/) has spent countless hours developing MABE2 and preparing for its release. Proper documentation and testing are essential for ensuring that MABE2 is accepted into and used by as large of a community as possible.
+​
+Write the Docs (link!), BLURB, puts the importance of proper documentation quite plainly:
+​
+> If people don’t know why your project exists,they won’t use it.
+> If people can’t figure out how to install your code, they won’t use it.
+> If people can’t figure out how to use your code, they won’t use it.
+​
+This is particularly important for the end-users of MABE2 who simply want to run experiments and do not want to waste time trying to understand the source code on their own, especially users who have less coding experience.
+​
+Documentation is also important for the MABE2 users who wish to write their own modules or otherwise contribute to this software. Write the Docs stresses the importance of documentation for potential contributors: 
+​
+> You only get contributions after you have put in a lot of work.
+> You only get contributions after you have users.
+> You only get contributions after you have documentation.
+​
+Software must also be tested before it is released to the community. End-users will (and should) be cautious of code that has no testing framework. Without code being tested how can we expect it to preform as expected? We must provide a testing suite to assure MABE2 users that this software runs correctly.
+​
+Testing frameworks are also helpful for future developers who want to make sure their contributions to this software does not break any previously functioning code.
```

Comment:
I think there's something wrong with your block quotes, as this entire section is one big block quote here https://github.com/athaichi/waves/blob/aria-lanea-blog/blog/_posts/2021-08-20-Aria-Lanea-Blog.md.